cairo-sys-rs/src/lib.rs

```diff
@@ -261,6 +265,12 @@ impl cairo_bool_t {
     }
 }
 
+impl From<bool> for cairo_bool_t {
+    fn from(b: bool) -> cairo_bool_t {
+        cairo_bool_t { value: b as _ }
```

Comment:
A `match` might be better, I don't think it's guaranteed that they're `true = 1`, `false = 0`.

Reply:
Or `b.to_glib()` maybe? :)

Reply:
No, glib support in cairo is optional.

Reply:
Ah crap, forgot that...

Reply:
Ok... I'd use an `if` here if that's ok with you...

Reply:
Sure, either way is fine :)